tools/gen-manifests-diff

```diff
@@ -61,9 +61,21 @@ def manifests_diff(tmp_path, rev):
 
 
 def main():
-    rev = "main"
-    if len(sys.argv) > 1:
-        rev = sys.argv[1]
+    match len(sys.argv):
+        case 1:
+            print("no revision given as argument, running against merge base")
+            # merge base is calculated against upstream
+            # "github.com/osbuild/images" not against the users fork
+            rev = subprocess.run(
+                ["git", "merge-base", "upstream/main", "HEAD"],
```

Comment:
This assumes the remote name, which might be different for every person.  Personally, I use `origin` for upstream and my name for my fork, for example.

It might be enough to just use `main`.  Assuming everyone starts every branch from a local copy of `main`, `git merge-base main HEAD` should be identical to `git merge-base upstream|origin/main HEAD`, if `upstream/main` is ahead of the local `main`.

Using plain `main` only becomes an issue (and would provide different results) if somehow the `main` has local modifications that don't exist upstream, but I think at that point, there's nothing we can do.

Reply:
Good point, I will close again for now then, if it becomes a problem we can make the code try to be smart and figure out how what remote name points to upstream